app/mailers/requests_confirmation_mailer.rb

```diff
@@ -25,6 +25,7 @@ def combined_items(request)
     # convert hash into an array of items with combined quantities
     grouped.map do |id, items|
       { 'item_id' => id, 'quantity' => items.map { |i| i['quantity'] }.sum }
+      { 'item_id' => id, 'quantity' => items.map { |i| i['quantity'] }.sum, 'name'=> Item.find(id).name }
```

Comment:
Looks like you meant to delete the line above this one?

Reply:
ARgh!